2_process.R

```diff
@@ -23,6 +24,35 @@ p2_targets_list <- list(
     format = "file"
   ),
   
+  # Reshape GFv1-NHDv2 xwalk table to return all COMIDs that drain to each PRMS segment
+  tar_target(
+    p2_drb_comids_all_tribs, 
+    p2_prms_nhdv2_xwalk %>%
+      select(PRMS_segid, comid_cat) %>% 
+      tidyr::separate_rows(comid_cat,sep=";") %>% 
+      rename(COMID = comid_cat)
+  ),
+  
+  # Process catchments so that every PRMS segment has >= 1 corresponding HRU; In addition,
+  # adjust catchments for 3 segments that were split in `delaware-model-prep` pipeline
+  # https://github.com/USGS-R/delaware-model-prep
+  tar_target(
+    p2_GFv1_catchments_edited_sf,
+    munge_GFv1_catchments(p1_GFv1_reaches_sf,p1_GFv1_catchments_sf,p2_drb_comids_all_tribs,GFv1_segs_split,crs_out = 4326)
+  ),
+  
+  # Save processed GFv1 catchments as a geopackage
+  tar_target(
+    p2_GFv1_catchments_edited_gpkg,
+    sf::st_write(p2_GFv1_catchments_edited_sf, 
+             dsn = "2_process/out/GFv1_catchments_edited.gpkg", 
+             layer = "GFv1_catchments_edited", 
+             driver = "gpkg",
+             quiet = TRUE,
+             # overwrite layer if already exists
+             append = FALSE)
+  ),
```

Comment:
```suggestion
             append = FALSE),
     format = 'file'
  ),
```

Since you are writing this locally, this target should saved as `format = 'file'` (right?)

Reply:
Good catch, thanks! I've created a new function `write_sf` (in `2_process/write_data.R`) that writes the sf object to a file and returns the file path so that we appropriately define `format = "file"` for this target.